src/main/resources/scss/pages/_tables.scss

```diff
@@ -0,0 +1,3 @@
+.jenkins-icon-size {
```

Comment:
`.jenkins` prefix shouldn't be used in the design library, its `jdl`

Reply:
is this even used though?

Reply:
The `t:iconsSize` internally uses this class and has a larger bottom margin. As the jdl-component-sample has a large padding those two would lead to bigger gap in the UI between table and sample code:
Without that it looks like this
![image](https://github.com/user-attachments/assets/9cec8060-52f8-46f8-a172-542488436cb9)

With margin removed:
![image](https://github.com/user-attachments/assets/591dd5f2-bfcd-4c3b-918c-042f96649306)

Found a way to avoid using it